Create CONDUCT.md (#4)
sjackman/talr: CONDUCT.md

# Contributing To The Project

If you decide to contribute anything, please do the following:

1. Fork this repository

> 1. Create a new branch from the latest `develop` (read more [here](https://guides.github.com/introduction/flow/))

Reviewer: ```suggestion
1. Create a new branch from the latest `master` (read more [here](https://guides.github.com/introduction/flow/))
```
We don't have a `develop` branch, but we do have a `master` branch.

Author: Sure, good call!

> 1. Start hacking on the new branch

Reviewer: ```suggestion
2. Start hacking on your new branch
```

> 1. Commit and push to the new branch

Reviewer: ```suggestion
3. Commit and push to your new branch
```

> 1. Make a pull request

Reviewer: ```suggestion
4. Open a pull request
```
